docs/source/external-tools/cdf/cdf_requirements.rst

```diff
@@ -161,7 +161,9 @@ This is a human readable description of the data variable. Generally, this is an
 
 ``DEPEND_0``
 ^^^^^^^^^^^^
-Explicitly ties a data variable to the time variable on which it depends. All variables which change with time must have a ``DEPEND_0`` attribute defined.
+Explicitly ties a data variable to the time variable on which it depends (i.e. - the `epoch` variable). All variables which change with time must have a ``DEPEND_0`` attribute defined. 
+
+Even if there is no time dependency of the data *within* the CDF file, SPDF has stated it would still be ideal to include a DEPEND_0 for each variable to ensure continuity *between* CDF files.  
```

Comment:
```suggestion
Even if there is no time dependency of the data *within* the CDF file, SPDF has stated it would still be ideal to include a ``DEPEND_0`` for each variable to ensure continuity *between* CDF files.  
```